Loop/FeedbackLoop.swift

```diff
@@ -52,31 +52,51 @@ extension FeedbackLoop {
     public struct Feedback {
         let events: (_ state: SignalProducer<State, Never>, _ output: FeedbackEventConsumer<Event>) -> Disposable
 
-        public init(
-            events: @escaping (
-            _ state: SignalProducer<State, Never>,
-            _ output: FeedbackEventConsumer<Event>
-            ) -> Disposable
+        /// Private designated initializer. See the public designated initializer below.
+        fileprivate init(
+            startWith events: @escaping (_ state: SignalProducer<State, Never>, _ output: FeedbackEventConsumer<Event>) -> Disposable
         ) {
             self.events = events
         }
 
         /// Creates a custom Feedback, with the complete liberty of defining the data flow.
         ///
-        /// - important: While you may respond to state changes in whatever ways you prefer, you **must** enqueue produced
-        ///              events using the `SignalProducer.enqueue(to:)` operator to the `FeedbackEventConsumer` provided
-        ///              to you. Otherwise, the feedback loop will not be able to pick up and process your events.
+        /// Consider using the standard `Feedback` variants, before deriving down to use this desginated initializer.
+        ///
+        /// Events must be explicitly enqueued using `SignalProducer.enqueue(to:)` with the `FeedbackEventConsumer`
+        /// provided to the setup closure. `enqueue(to:)` respects producer cancellation and removes outstanding events
+        /// from the loop internal event queue.
+        ///
+        /// This is useful if you wish to discard events when the state changes in certain ways. For example,
+        /// `Feedback(skippingRepeated:effects:)` enqueues events inside `flatMap(.latest)`, so that unprocessed events
+        /// are automatically removed when the inner producer has switched.
+        ///
+        /// - important: The `state` producer provided to the setup closure **does not** replay the current state.
```

Comment:
Hm i wondering why do they use [replay subject](https://github.com/NoTests/RxFeedback.swift/blob/master/Sources/RxFeedback/ObservableType%2BRxFeedback.swift#L35) in Rx 🤔

Reply:
That's interesting. I can look at whether we can make it provide replay-1 semantic.